contrib/lang/erlang-elixir/packages.el

```diff
@@ -21,6 +22,41 @@
   "List of all packages to install and/or initialize. Built-in packages
 which require an initialization must be listed explicitly in the list.")
 
+(defun erlang-elixir/init-alchemist ()
+  (add-hook 'elixir-mode-hook 'alchemist-mode)
+  (setq alchemist-project-compile-when-needed t)
+  (evil-leader/set-key-for-mode 'elixir-mode
+    "mmd" 'alchemist-mix-execute
+    "mta" 'alchemist-mix-test
+    "mtb" 'alchemist-mix-test-this-buffer
+    "mtf" 'alchemist-mix-test-file
```

Comment:
Conventions tell to avoid the distinction between buffers and files and to define only `mtb`.
You should remove `mtf` and create a function to auto-save the file before executing the tests with `mtb`.